lib/tracker.ex

```diff
@@ -19,6 +19,11 @@ defmodule Tracker do
     end)
   end
 
+  def complete? do
+    {total, completed} = Agent.get(__MODULE__, &(&1))
+    total == Enum.count(completed)
```

Comment:
In my previous implementation, I actually kept a full list of the modules rather than just the count. For some reason, this worries me slightly that we might accidentally end up counting the wrong thing or something. Any thoughts on that?